sound/soc/intel/skylake/skl.c

```diff
@@ -794,7 +794,7 @@ static void skl_probe_work(struct work_struct *work)
 {
 	struct skl_dev *skl = container_of(work, struct skl_dev, probe_work);
 	struct hdac_bus *bus = skl_to_bus(skl);
-	struct hdac_ext_link *hlink = NULL;
+	struct hdac_ext_link *hlink;
```

Comment:
need to split those patches in two

ASoC: codecs: hdac_hdmi: 

and 

ASoC: Intel: Skylake:...

Reply:
done